Automattic/wp-calypso
# Social previews | Fix/Improve styling

Fixes 1203820933396971-as-1204619418011130/f

## Proposed Changes

* This PR fixes the styling for the post sharing modal at `/posts/:site`
* It also unifies the styling for Social Preview sections to make them consistent
* It makes the social previews responsive for smaller screen sizes

## Testing Instructions

- Make sure you have a self-hosted site connected to Jetpack, with a subscription that includes Social.
- Spin up Calypso, by running this branch locally or using a live link below.
- Ensure that you have Social Media accounts connected for a self-hosted WP Site
- Go to `/posts/:site` for a self-hosted WP site or a JN site
- Click on the 3 dots for any post and Click "Share"
- Click on "Preview"
- Confirm that there is no duplicate scrollbar in the modal
- Scroll on the modal header
- Confirm that it does not scroll
- Confirm that the modal content is scrollable as before

Here is the demonstration of the BEFORE and AFTER behavior

https://github.com/Automattic/wp-calypso/assets/18226415/bde7be68-5a7c-4d9c-904d-de4250368388


Mobile view

| Before | After |
|--------|--------|
| <img width="340" alt="Screenshot 2023-05-19 at 1 15 54 PM" src="https://github.com/Automattic/wp-calypso/assets/18226415/5b31afc8-79e8-4963-8575-46ccbf906435"> | <img width="340" alt="Screenshot 2023-05-19 at 1 16 11 PM" src="https://github.com/Automattic/wp-calypso/assets/18226415/add36e88-eb76-44fb-8035-18e10a53ded1"> |


## Pre-merge Checklist




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1204619418011130
